E/eudev/build_tarballs.jl

```diff
@@ -3,17 +3,25 @@
 using BinaryBuilder, Pkg
 
 name = "eudev"
-version = v"3.2.9"
+version = v"3.2.14"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://dev.gentoo.org/~blueness/eudev/eudev-3.2.9.tar.gz", "89618619084a19e1451d373c43f141b469c9fd09767973d73dd268b92074d4fc")
+    GitSource("https://github.com/eudev-project/eudev", "9e7c4e744b9e7813af9acee64b5e8549ea1fbaa3")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/eudev*
-apk add gperf
+apk add gperf autoconf automake libtool libxslt-dev docbook-xsl
+./autogen.sh
+
+# Only apply the patch for musl targets
+if [[ "${target}" == *"musl"* ]]; then
+    echo "Applying musl-specific thread_local fix"
+    find . -name "*.c" -exec grep -l "thread_local" {} \; | xargs -r sed -i 's/thread_local/__thread/g'
+fi
```

Comment:
Can we get a real patch instead of some sed magics?

Reply:
And please don't use [gen AI tools to generate a made up patch](https://github.com/JuliaLang/julia/pull/58628#discussion_r2125092176)